modules/n1ql/pages/n1ql-language-reference/insert.adoc

```diff
@@ -1280,13 +1289,12 @@ EXPLAIN INSERT INTO `travel-sample` (key UUID(), value _airport)
           "#operator": "Sequence",
           "~children": [
             {
-              "#operator": "IntersectScan", // <3>
+              "#operator": "IntersectScan",
               "scans": [
                 {
-                  "#operator": "IndexScan3", // <2>
-                  "as": "_airport",
-                  "index": "def_airportname",
-                  "index_id": "af738c1d1aa1f107",
+                  "#operator": "IndexScan3",
```

Comment:
Make sure to add the callout numbers back into the examples when you update them
```suggestion
                  "#operator": "IndexScan3", // <2>
```